packages/quantic/force-app/examples/main/lwc/actionSendRating/actionSendRating.js

```diff
@@ -9,7 +9,7 @@ export default class ActionSendRating extends LightningElement {
     }, 0);
   }
   handleSendRating() {
-    const customEvent = new CustomEvent('rating', {
+    const customEvent = new CustomEvent('quantic__rating', {
```

Comment:
I think the event names in the examples/ I would leave alone

Reply:
will revert for examples

Reply:
Did we decide to revert the events in the */examples as they are not part of the package? @SimonMilord